recipes-overlayed/kselftests/files/0001-selftests-bpf-Adding-config-fragment-CONFIG_CGROUP_B.patch

```diff
@@ -0,0 +1,26 @@
+From 63060c39161d3d61c771dee20a3cbdffaf83f1df Mon Sep 17 00:00:00 2001
+From: Naresh Kamboju <naresh.kamboju@linaro.org>
+Date: Tue, 12 Dec 2017 00:55:23 +0530
+Subject: [PATCH 1/8] selftests: bpf: Adding config fragment
+ CONFIG_CGROUP_BPF=y
+
+CONFIG_CGROUP_BPF=y is required for test_dev_cgroup test case.
+
+Signed-off-by: Naresh Kamboju <naresh.kamboju@linaro.org>
+Signed-off-by: Daniel Borkmann <daniel@iogearbox.net>
+---
```

Comment:
Missing Upstream-Status: tag

Reply:
FYI,
The maintainer tree shows it is accepted.
selftests: bpf: Adding config fragment CONFIG_CGROUP_BPF=y
https://patchwork.ozlabs.org/patch/847197/